_posts/2025-04-06-Hite-StreetComplete.md

```diff
@@ -0,0 +1,42 @@
+
+# Finding an issue
+
+StreetComplete has a fairly simple way to track issues and proposed changes. However it appears that the issue tracker entries for new features and any bugs appear to take a at least three weeks to get someone to start working on a bug and usually longer to add a new feature. 
+
+Since December 2023 it appears that most of the work on StreetComplete has been focused on producing an IOS version of StreetComplete. Because of all of this focus on producing an IOS app and whatever remaining effort on the program is going to focusing on the bug reports and feature requests there has been little to no work on updating the docs for StreetComplete.
+
+There are several spots in the wiki where a sentence could use reworking to make it more readable or information was out of date and needed to be updated.
+
+## Updating "Some quests are obstructed by other quests, and I can't zoom in enough" so that the information is up to date
+![issue1](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-134647.png)
+The original issue seen here used language like "when ... 59.0 is released" whenever the current version is 60.3. I made sure to update the language so that it uses the correct tense and includes information about 59.0 being released.
+
+![issue1Fixed](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-153024.png)
+
+## Updating "Can I choose photos from the gallery for uploading to notes, instead of using camera?" so the language is clearer and can be understood easier
+![issue2](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-144006.png)
+The screenshot seen here shows a few sentences that are not very clear. The original version appeared to be omiting several words that made it seem almost jumpy in its sentences as though it was missing something. I added in some language that should help make it feel smoother and the sentences flow together better.
+
+![issue2Fixed](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-160538.png)
+
+## Updating "When I click on quest, several similar icons pop up, why?" 
+![issue3](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-161933.png)
+The photo shown here shows another example of the FAQ answers not being worded quite right. The sentence shown here appeared to be nearly unintelligble. It took me five times as long to understand what this was trying to say then it did for me to rewrite the answer.
+
+![issue3Fixed](https://github.com/ah1034/hfoss2025-blogs/blob/newBranch/assets/images/ah1034/Screenshot-2025-04-06-163349.png)
```

Comment:
```suggestion
![issue3Fixed](/hfoss2025-blogs/assets/images/ah1034/Screenshot-2025-04-06-163349.png)
```
You dont need to use the full github URL (which didnt seem to be working for me anyway), a path like this should work for referencing the images that you uploaded to the directory when the site is built.

Could you update all the image paths to be like this suggestion?